crates/pathfinder/src/config/builder.rs

```diff
@@ -93,6 +93,20 @@ impl ConfigBuilder {
             }
             None => std::num::NonZeroUsize::new(2).unwrap(),
         };
+        let enable_sqlite_wal = match self.take(ConfigOption::EnableSQLiteWriteAheadLogging) {
+            Some(enable) => {
+                let enable = enable.to_lowercase();
+                match enable.as_str() {
+                    "on" | "true" | "yes" => Ok(true),
+                    "off" | "false" | "no" => Ok(false),
```

Comment:
One alternative is:
```
sqlite-wal = enable | disable (or on | off)
```
and only accept that.

Reply:
I'd vote for `on | off`. I'll remove the other variants.

Reply:
Or not? `--enable-sqlite-wal off` seems awkward.

Reply:
Maybe `true | false` would suffice?

Reply:
I've updated the code to accept `true | false`.